android/src/main/java/com/inbrain/InBrainSurveysModule.java

```diff
@@ -94,46 +94,35 @@ public void setDataOptions(final ReadableMap data) {
     }
 
 
-    // ************************x
-    // ***** OPEN WALL ******
-    // ************************
-
-    // TODO: - Move the initFromRaw to Android SDK
-    WallOption mapRawToOption(Integer rawValue) {
-        switch (rawValue) {
-            case 0:
-                return WallOption.ALL;
-            case 1:
-                return WallOption.SURVEYS;
-            case 2:
-                return WallOption.OFFERS;
-        }
-
-        return WallOption.ALL;
-    }
-
+// ************************
+// ***** OPEN WALL ******
+// ************************
     @ReactMethod
     public void openWall(final int option, final Promise promise) {
-        WallOption wallOption = mapRawToOption(option);
+    WallOption wallOption = WallOption.fromRaw(option);
```

Comment:
That does not work. Pls, check the comment for Flutter SDK.
And pls, run the app before asking about the review

Reply:
Done